### PR TITLE
[BTAT-343] Added Service Info banner with BTA links and User Name

### PR DESCRIPTION
**Pull request by MJCallahanPage**

### CODE REVIEW

File: test/assets/TestConstants.scala [hunk 1]
```diff
@@ -19,18 +19,27 @@ package assets
...
 import utils.ImplicitDateFormatter
 
 object TestConstants extends ImplicitDateFormatter {
 
   val testMtditid = "XAIT0000123456"
   val testNino = "AB123456C"
-  val testMtdItUser: MtdItUser = MtdItUser(testMtditid, testNino)
+  val testUserName = "Albert Einstein"
+  val testUserDetails = UserDetailsModel(testUserName, None, "n/a", "n/a")
+  val testUserDetailsUrl = "/user/oid/banana"
```
Comment (sraspin) [inline test/assets/TestConstants.scala]: Shouldn't this be potato?!

File: test/views/helpers/renderServiceInfoHelperSpec.scala [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          manageAccountLink.text() shouldBe messages.btaManageAccount
+        }
+
+        s"should have a link to '${appConfig.businessTaxAccount}'" in {
+          manageAccountLink.attr("href") shouldBe appConfig.btaManageAccountUrl
+        }
+
+      }
+
+      "have a link to Manage Aaccount" which {
```
Comment (sraspin) [inline test/views/helpers/renderServiceInfoHelperSpec.scala]: minor change: should be 'messages'

File: test/assets/TestConstants.scala (hunk 1 above)
Comment (MJCallahanPage) [inline test/assets/TestConstants.scala]: :smiley_cat:

File: test/views/helpers/renderServiceInfoHelperSpec.scala (hunk 2 above)
Comment (MJCallahanPage) [inline test/views/helpers/renderServiceInfoHelperSpec.scala]: Correcting :)

**Pull request merged**